containers/python-3-anaconda-postgres/README.md

```diff
@@ -0,0 +1,166 @@
+
+```yaml
+args:
+  NODE_VERSION: "14" # Set to "none" to skip Node.js installation
+```
+
+#### Installing a different version of Python
+
+As covered in the [user FAQ](https://docs.anaconda.com/anaconda/user-guide/faq) for Anaconda, you can install different versions of Python than the one in this image by running the following from a terminal:
+
+```bash
+conda install python=3.6
+```
+
+Or in a Dockerfile:
+
+```Dockerfile
+RUN conda install -y python=3.6
+```
+
+### [Optional] Adding the contents of environment.yml to the image
+
+For convenience, this definition will automatically install dependencies from the `environment.yml` file in the parent folder when the container is built. You can change this behavior by altering this line in the `.devcontainer/Dockerfile`:
+
+```Dockerfile
+RUN if [ -f "/tmp/conda-tmp/environment.yml" ]; then /opt/conda/bin/conda env update -n base -f /tmp/conda-tmp/environment.yml; fi \
+    && rm -rf /tmp/conda-tmp
+```
+
+### Adding the definition to your folder
+
+1. If this is your first time using a development container, please see getting started information on [setting up](https://aka.ms/vscode-remote/containers/getting-started) Remote-Containers or [creating a codespace](https://aka.ms/ghcs-open-codespace) using GitHub Codespaces.
+
+2. Start VS Code and open your project folder or connect to a codespace.
+
+3. Press <kbd>F1</kbd> select and **Add Development Container Configuration Files...** command for **Remote-Containers** or **Codespaces**.
+
+   > **Note:** If needed, you can drag-and-drop the `.devcontainer` folder from this sub-folder in a locally cloned copy of this repository into the VS Code file explorer instead of using the command.
+
+4. Select this definition. You may also need to select **Show All Definitions...** for it to appear.
+
+5. Finally, press <kbd>F1</kbd> and run **Remote-Containers: Reopen Folder in Container** or **Codespaces: Rebuild Container** to start using the definition.
+
+## Testing the definition
+
+This definition includes some test code that will help you verify it is working as expected on your system. Follow these steps:
+
+1. If this is your first time using a development container, please follow the [getting started steps](https://aka.ms/vscode-remote/containers/getting-started) to set up your machine.
+2. Clone this repository.
+3. Start VS Code, press <kbd>F1</kbd>, and select **Remote-Containers: Open Folder in Container...**
+4. Select the `containers/python-3-anaconda-postgres` folder.
+5. After the folder has opened in the container, press <kbd>F5</kbd> to start the project.
+6. You should see `DATABASE CONNECTED` in the debug console and a `test-project/plot.png` file should be added to the folder after it runs with the plot result.
```

Comment:
Now that database.py is all we run via `F5-debug`  in _launch.json_, we'll want to change this text, or allow debuging of both _test-project/*.py_ samples